inc/command/class-command.php

```diff
@@ -188,14 +202,224 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		return $this->run_command( $input, $output, 'vendor/bin/phpunit', $options );
 	}
 
+	/**
+	 * Runs Codecept with zero config by default.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function codecept( InputInterface $input, OutputInterface $output ) {
+		$options = $input->getArgument( 'options' );
+		$module = $input->getOption( 'module' );
+		$run_headless_browser = $input->getOption( 'browser' );
+		$use_chassis = $input->getOption( 'chassis' );
+		$tests_folder = $module !== 'project' ? "altis/$module/tests" : 'tests';
+		$project_subdomain = $this->get_project_subdomain();
+
+		// Write the default config.
+		$config = [
+			'paths' => [
+				'tests' => $tests_folder,
+				'output' => 'altis/dev-tools/tests/_output',
```

Comment:
I think putting outputs in the package runs of the risk of hiding them and might cause composer some issues when trying to update if it detects modifications to the package.

The default setup is to use the `tests` directory so I think all of these are intended to be colocated with the tests e.g.
```suggestion
				'output' => '../tests/_output',
```

You'd have to check the directories exist and create them before running codeception if not.